be/src/exec/hdfs_file_reader.cpp

```diff
@@ -80,7 +80,7 @@ Status HdfsFileReader::open() {
     _hdfs_file = hdfsOpenFile(_hdfs_fs, _path.c_str(), O_RDONLY, 0, 0, 0);
     if (_hdfs_file == nullptr) {
         std::stringstream ss;
-        ss << "open file failed. " << _namenode << _path;
+        ss << "open file failed. " << _namenode << _path << ", err: " << strerror(errno);;
```

Comment:
Also add BE ip info so that we can easily find which BE report the error.
You can call `BackendOptions::get_localhost();`